.env.example

```
@@ -1,7 +1,11 @@
-# fetcher가 실제로 보는 값으로 백엔드 render 주소 사용
-NEXT_PUBLIC_SOCKET_URL= 
-NEXT_PUBLIC_API_BASE_URL=
\ No newline at end of file
+# API base url
+NEXT_PUBLIC_API_BASE_URL=
+
+# frontend base url
+NEXT_PUBLIC_BASE_URL=
+
+# socket server url
+NEXT_PUBLIC_SOCKET_URL=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)